meta.yaml

```diff
@@ -0,0 +1,42 @@
+
+{% set name = "FlaGs" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/chayan7/FlaGs/archive/v1.2.6.tar.gz
+  sha256: 0d64fa8a24ad1d79b9ea570549bf2278cd899ab15f65f63b5b65d7420a229f7d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install biopython==1.72 ete3 hmmer -vv"
```

Comment:
you shouldn't use `pip` to install dependencies. Dependencies need to be listed under the `requirements` section. `pip` is used to actually install this package once conda has installed all dependencies. But it doesn't look like your package is `pip` installable since it lacks a `setup.py` or any of the other build systems. Therefore you would need to use a `build.sh`/`build.bat` to copy the necessary files into the directory specified by the `${PREFIX}` environment variable.